Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -25,13 +26,25 @@ public function fix(\SplFileInfo $file, $content)
     {
         // [Structure] A file must always end with a linefeed character
 
-        $content = rtrim($content);
+        if (empty($content)) {
+            return $content;
+        }
+
+        $tokens = Tokens::fromCode($content);
+        $count = $tokens->count();
+        if (0 === $count) {
+            return;
```

Comment:
I guess that solves it then :) i.e. the test is testing a scenario that can never happen and can therefor be removed.... or do I miss something?